batch7crud-roll1/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -99,16 +103,23 @@ private void create(HttpServletRequest request, HttpServletResponse response) th
 
   private void createProcess(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 
-    String userName = request.getParameter(USER_NAME);
-    String password = request.getParameter(PASSWORD);
-    String fullName = request.getParameter(FULL_NAME);
-    String department = request.getParameter(DEPARTMENT);
-    String address = request.getParameter(ADDRESS);
-    Employee employee = new Employee(userName, password, fullName, department, address);
+    Map<String, String> inputs = new HashMap<>();
+    inputs.put(USER_NAME, request.getParameter(USER_NAME));
+    inputs.put(PASSWORD, request.getParameter(PASSWORD));
+    inputs.put(FULL_NAME, request.getParameter(FULL_NAME));
+    inputs.put(DEPARTMENT, request.getParameter(DEPARTMENT));
+    inputs.put(ADDRESS, request.getParameter(ADDRESS));
+    inputs.put(AGE, request.getParameter(AGE));
 
     try {
+      EmployeeFactory employeeFactory = new EmployeeFactory();
+      Employee employee = employeeFactory.getEmployee(inputs);
       employeeService.create(employee);
       response.sendRedirect(request.getContextPath() + EMPLOYEE_LIST);
```

Comment:
are you showing success message once the user gets successfully added to the db ?